src/core/shards/casting.cpp

```diff
@@ -513,8 +320,8 @@ template <SHType ET> struct ExpectX {
   }
 };
 
-#define EXPECT_SHARD(_name_, _shtype_)                 \
-  struct Expect##_name_ : public ExpectX<_shtype_> {}; \
+#define EXPECT_SHARD(_name_, _cbtype_)                 \
+  struct Expect##_name_ : public ExpectX<_cbtype_> {}; \
```

Comment:
Is that a failed merge conflict resolution? I believe all mention of "cb" should have been replaced by "sh".

Reply:
yep 👍